Resources/public/js/ezrecommendation.js

```diff
@@ -0,0 +1,77 @@
+// register HandleBars helpers when library is loaded
+
+document.addEventListener('DOMContentLoaded', function() {
+    Handlebars.registerHelper('each', function(context, options) {
```

Comment:
AFAIK, `each` is a built-in helper in Handlebars, there's no need to redefine it: http://handlebarsjs.com/block_helpers.html#iterators